docs/deprecatednoderuntime.md

```
@@ -0,0 +1,81 @@
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4        # runs.using: node20
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20               # EOL build target
+```
+
+sisakulint reports:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)